apps/src/templates/lessonOverview/StudentLessonOverview.jsx

```diff
@@ -0,0 +1,180 @@
+import PropTypes from 'prop-types';
+import React, {Component} from 'react';
+import i18n from '@cdo/locale';
+import {announcementShape} from '@cdo/apps/code-studio/announcementsRedux';
+import Announcements from '../../code-studio/components/progress/Announcements';
+import {connect} from 'react-redux';
+import {SignInState} from '@cdo/apps/templates/currentUserRedux';
+import {ViewType} from '@cdo/apps/code-studio/viewAsRedux';
+import SafeMarkdown from '@cdo/apps/templates/SafeMarkdown';
+import InlineMarkdown from '@cdo/apps/templates/InlineMarkdown';
+import styleConstants from '@cdo/apps/styleConstants';
+import color from '@cdo/apps/util/color';
+import LessonNavigationDropdown from '@cdo/apps/templates/lessonOverview/LessonNavigationDropdown';
+import {reducedLessonShape} from '@cdo/apps/templates/lessonOverview/lessonPlanShapes';
+
+const styles = {
+  frontPage: {
+    display: 'flex',
+    flexDirection: 'row',
+    marginTop: 40
+  },
+  left: {
+    width: '60%',
+    paddingRight: 20
+  },
+  right: {
+    width: '40%',
+    padding: '0px 10px 10px 20px',
+    borderLeft: 'solid 1px #333'
+  },
+  header: {
+    margin: '10px 0px',
+    display: 'flex',
+    justifyContent: 'space-between'
+  },
+  navLink: {
+    fontSize: 14,
+    lineHeight: '22px',
+    color: color.purple,
+    margin: '10px 0px'
+  },
+  copyResourceWarningArea: {
+    color: '#8a6d3b',
```

Comment:
Consider using a color from the util?

Reply:
Same with backgroundColor

Reply:
oops these are actually all no longer necessary, I'll take them out